source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service.py

```diff
@@ -81,9 +81,11 @@ def _call_service(self, params, headers={}):
             self.call_params += '{}={}&'.format(key, value)
 
         try:
-            return self.breaker.call(
-                requests.get, url=self.service_url, headers=headers, params=params, timeout=self.timeout
+
+            toto = self.breaker.call(
+                requests.get, url=self.service_url, headers=headers, params=params, timeout=self.timeout, verify=verify
             )
+            return toto
```

Comment:
`toto`? Maybe you can inline like it was before.

Reply:
Sorry it's my bad